### ISSUE TITLE
Add guide to setup remote access to private networks

### ISSUE THREAD

**Issue by gdemonet** [scality/metalk8s#2187]
**Component**: docs

**Why this is needed**:

Recommended architectures use private networks for the cluster control and workload planes. By definition, the host will need some form of setup to access these networks (e.g. using a VPN).

**What should be done**:

Since accessing cluster services from the user's host is a common task, it would make sense to provide minimal guidance on how to setup such a VPN in various contexts (with a well-known cloud provider, using OpenStack routers, on bare-metal with `sshuttle`, ...).

**Comment by NicolasT** [comment]
I disagree.

Whilst we can document something in scope of e.g., running a test cluster on OpenStack, then explaining how to access such 'private networks', I don't think documenting something *in general* is right: the networks used for control-plane and workload-plane are *existing* networks in a datacenter in which a cluster is deployed, and how to access these networks depends 100% on the datacenter network topology and such. However, we can *expect* that any node which needs access to some service provided by the cluster (this may be an administrator's machine accessing the control-plane network, or some application servers accessing services on the workload-plane) is already in place (using whatever relevant means).

So, if the scope of this request is dev/test environments on e.g., OpenStack, then sure, in the 'developer guide'. Not in the end-user docs.

**Comment by gdemonet** [comment]
Fair point, could be part of the `contrib/terraform/README` for a start, and maybe also add a section for this in the developer docs.
